presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8247,6 +8248,53 @@ public void testRightJoinWithEmptyProbeSide()
                 "WITH small_part AS (SELECT * FROM part WHERE name = 'a') SELECT lineitem.orderkey FROM small_part RIGHT JOIN lineitem ON  small_part.partkey = lineitem.partkey");
     }
 
+    @Test
+    public void CardinalityOfFilterSimplification()
```

Comment:
I think for tests you would have to turn on the session property.

Reply:
I would recommend add separate test similar to `TestSimplifyExpressions` to make sure the expression converted to the desired expression.

Reply:
The session property is on by default. Don't we use the default configuration/properties for testing?